doc/README.Win32.md

```diff
@@ -174,6 +174,18 @@ with control-S.
 Visual Studio will then re-run CMake.  If that completes without errors,
 you can build with Build > "Build All".
 
+### Building libpcap - Using vcpkg ###
```

Comment:
Is this *building* libpcap (i.e., is this along the line of various UN\*X package managers that provide pre-packaged source and build instructions), or is it installing a pre-packaged binary of libpcap (i.e., is this along the line of various U\*X package managers that install a prebuilt binary package)?

Reply:
Sorry to reply you so late, yes, it's the first case you mentioned.`is this along the line of various UN*X package managers that provide pre-packaged source and build instructions`

Reply:
npcap is not currently in vcpkg, you can submit an issue in vcpkg or directly submit a PR to add a new port, so that we will make libpcap depend on npcap and will realize the packet capture function.